CONTRIBUTING.md

```diff
@@ -202,20 +237,45 @@ When opening a pull request, please use at least one of the following prefixes:
 * **[ENH]** for enhancements
 * **[FIX]** for bug fixes
 * **[REF]** for refactoring existing code
-* **[STY]** for stylistic changes
 * **[TST]** for new or updated tests, and
-* **[WIP]** for changes which are not yet ready to be merged
+* **[MAINT]** for maintenance of code
+
+You can also combine the tags above, for example if you are updating both a test and
+the documentation: **[TST, DOC]**.
 
 Pull requests should be submitted early and often!
-If your pull request is not yet ready to be merged, please also include the **[WIP]** prefix.
+If your pull request is not yet ready to be merged, please use [draft PRs][link_draftpr]
 This tells the development team that your pull request is a "work-in-progress",
 and that you plan to continue working on it.
-We request that you do not use the Draft PR feature at this time, 
-as it interferes with our Continuous Integration tool, Travis.
+Note that if your pull request has no conversation or commits for 90 days,
+stale-bot will mark it stale.
+We may also ask that another development team member resume work on it, if you are unable to continue to do so.
+
+### Pull Request Checklist (Before Requesting Review):
+- [ ] Check that all tests are passing ("All tests passsed")
+- [ ] Make sure you have docstrings for any new functions
+- [ ] Make sure that docstrings are updated for edited functions
+- [ ] Make sure you note any issues that will be closed by your PR
+- [ ] Take a look at the automatically generated readthedocs for your PR (Show all checks -> continuous-documentation/readthedocs -> Details)
+
+### After Changes Are Merged
+After successful merging of the pull request, remember to [keep your fork up to date][link_updateupstreamwiki] with the master `tedana` repository and to delete the branch on your fork that was used for the merged pull request.
+You may also want to update your other branches to include these changes,
```

Comment:
I think this is redundant with the idea of merging in master, which I'd again encourage folks not to use unless there's a merge conflict.

Reply:
Agreed. Since we're suggesting as a first step in the contribution process that users fetch & merge upstream/master (before creating a new branch), I think we can safely remove this section!

Reply:
Okay, I've just deleted those sections.